dataduct/steps/etl_step.py

```diff
@@ -452,11 +452,11 @@ def pop_inputs(input_args):
         return input_args
 
     @staticmethod
-    def get_output_s3_path(output_path):
+    def get_output_s3_path(output_path, is_directory=False):
```

Comment:
can you default this to true ? As I think it is an issue with everystep and not just transform and that would require no changes to the steps.